### PR TITLE
Add initial Gitpod yml file

### PR DESCRIPTION
**Pull request by jwmoore1**
## Description
Create an initial process to use [Gitpod](https://gitpod.io/) with AZQS so PRs can be created in the cloud.

## Types of changes
Adding a .gitpod.yml file.

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [X] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
+image: devwithlando/gitpod:1
+
+tasks:
+  - name: Setup Drupal Dev Env
+    #init: |
+    # docker build -t devwithlando/php:8.0-apache-4 .
+    # docker build -t devwithlando/phpmyadmin:5.1.1 .
```
Comment (joeparsons) [inline .gitpod.yml:8]:
```suggestion
```

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
+image: devwithlando/gitpod:1
+
+tasks:
+  - name: Setup Drupal Dev Env
```
Comment (joeparsons) [inline .gitpod.yml:5]:
```suggestion
  - name: Setup Quickstart Dev Env
```

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    onOpen: ignore
+  - port: 443
+    onOpen: ignore
+
+github:
+  prebuilds:
+    master: true
+    branches: false
+    pullRequests: true
+    pullRequestsFromForks: true
```
Comment (trackleft) [inline .gitpod.yml:24]:
```suggestion
    pullRequestsFromForks: false
```

**Comment by joeparsons** [comment]
Per 2023-08-09 meeting discussion, we're going to "archive" this and probably open a new PR when we have time to re-visit.

**Pull request closed without merging**